### ISSUE TITLE
Missing stem from intraannual dendro survey

### ISSUE THREAD

**Issue by jenajordan** [SCBI-ForestGEO/Dendrobands#117]
Stem #60582 is not on the dendro survey list I don't believe, but I came across it last week on the plot. It has an auto dendrometer on it. We may also want to confirm that all (7 or 8) auto dendrometer stems are on the survey list.

**Comment by rudeboybert** [comment]
Good catch @jenajordan. What had happened was in running database of stems we build all the field forms from [scbi.dendroAll_BLANK.csv](https://github.com/SCBI-ForestGEO/Dendrobands/blob/master/data/scbi.dendroAll_BLANK.csv)
1. I had set all 8 autodendrometer stems to be biweekly
2. Then I did the rebalancing of biweekly/biannual stems in the [Google Sheet](https://docs.google.com/spreadsheets/d/1rneieQOCclZ2q-Kbxog-rzNMM6-9d7foGooTv8Xq588/edit#gid=1289938519).

As a result, two of the autodendrometer stems got sent back to being biannual only.  I should've done steps 1 and 2 above in the reverse order.

Heads up @teixeirak: b/c of this fix we now have 152 stems in the biweekly survey instead of 150. I think is this tolerable
